services/net/scheduler/SchedulerManager.cs

```diff
@@ -73,8 +73,8 @@ public override async Task RunAsync()
                             else if (ev.EventType == EventScheduleType.Notification)
                                 await GenerateNotificationRequestAsync(scheduledEvent);
 
-                            // Update even last ran on
-                            scheduledEvent.LastRanOn = DateTime.UtcNow;
+                            // Update event request sent on.
+                            scheduledEvent.RequestSentOn = DateTime.UtcNow;
```

Comment:
The schedule uses this date time value to ensure it doesn't keep sending requests to the reporting service.